content/docs/use-cases/ci-and-ml.md

```diff
@@ -0,0 +1,172 @@
+# CI and ML
+
+## Motivation
+
+ML engineers and data scientists face many unique challenges. Some of these
+challenges can be solved by adopting well-established concepts from traditional
+software engineering and development. Probably the most common adoption is
+versioning control. Another adoption, unfortunately less common though, could be
+unit testing the various stages of a data processing pipeline. Here is a non
+exhaustive list of a few other unique challenges:
+
+- automatically re-train models upon changes
+- systematically compare (and report) performance of different versions of a
+  model
+- test data correctness when new datasets are introduced
+- handle a model training that is computationally intensive
```

Comment:
No capitalization nor punctuation. @jorgeorpinel anything else here?

Reply:
Capital letters are optional indeed according to our guidelines but we don't really use this format anywhere. No worries, I'll update it for you bellow so we can move on:

```suggestion
versioning control. Another adoption, unfortunately less common though, could be
unit testing the various stages of a data processing pipeline. Here is a non
exhaustive list of a few other unique challenges:

- Automatically re-train models upon changes
- Systematically compare (and report) performance of different versions of a
  model
- Test data correctness when new datasets are introduced
- Handle a model training that is computationally intensive
```